back/app/services/idea_custom_fields_service.rb

```diff
@@ -70,6 +70,8 @@ def allowed_extra_field_keys
         fields_with_array_keys[field.key.to_sym] = []
       when 'file_upload'
         fields_with_array_keys[field.key.to_sym] = %i[content name]
+      when 'point'
```

Comment:
After we discussed this in a call, we decided this may be a bad idea, since `'point'` input_type also used by `custom_fields` with `code: 'location_point_geojson'`, and `'point'` is sufficiently descriptive as it is.